notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>i chose this example target because it was listed as a false positive ?</p><p> </p>

Reply:
I think the point may be moot, since we chatted about using a different target.

That said, even though follow-up observations have shown this to be a false positive, it is interesting that our period differs from the pipeline prediction. It raises questions about what they might have done differently from our analysis

Reply:
<p>For the record, I think we can pass on this for now</p>